dev-manuals/test-plans/template.md

```diff
@@ -1,32 +1,31 @@
-# Name (`Test Case ID`)
+# $CATEGORY: $ACTION (`$TESTCASE_ID im Format "u.$(kebab-case $CATEGORY)-$NUMBER (from 1)"`)
 
-Test Case name pattern: u.XY(service identifier)-Z (sequential number, start at 1).  
-Version: vx.x, date of creation, description of change  
-Author: Author of the use case description  
-Tester: Use case tester
+Version: `v$X.$Y with $X >= 1, $Y >= 0`, `$NOW in YYYY-mm-dd`, `$CHANGE_DESCRIPTION like a git commit` \
+Author: Authors of this file, comma-separated
+(Delete this line in test cases!) Filename: `$(kebab-case u-$CATEGORY-$NUMBER-$ACTION).md`
```

Comment:
To what is `(Delete this line in test cases!)` referenced to? Shouldn't be there line breaks?

Reply:
The following line with the filename.
That line is simply to clairfy how to write the filename.